### PR TITLE
heksa 1.12.1 (new formula)

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Just experimenting with #56553 - trying to resolve test issues.

### CODE REVIEW

File: Formula/heksa.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Heksa < Formula
+  desc "CLI hex dumper with colors"
+  homepage "https://github.com/raspi/heksa"
+  url "https://github.com/raspi/heksa.git",
+      :tag      => "v1.12.1",
+      :revision => "54a68a2910a87b8b9cfcda28b8693e18dd9e2463"
```
Comment (vladimyr) [inline Formula/heksa.rb:6]: This is intentional because as part of the build process git revision gets queried.

File: Formula/heksa.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  def install
+    system "make", "build"
+    bin.install "bin/heksa"
+  end
+
+  test do
+    require "pty"
+
+    cp test_fixtures("test.png"), testpath/"test.png"
```
Comment (chenrui333) [inline Formula/heksa.rb:19]: Interesting, why do we need this step?

Comment (vladimyr) [reply Formula/heksa.rb:19]: To make command shorter 😀
It probably isn't needed, that's leftover from my previous experimentation and I'm lazy enough to leave it there 😀

Comment (chenrui333) [reply Formula/heksa.rb]: 😆

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@vladimyr, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**